static/styles/portico/portico.scss

```diff
@@ -1012,6 +1012,10 @@ input#terminal:checked ~ #tab-terminal {
     margin-left: 10px;
 }
 
+#register-button {
```

Comment:
`#registration .register-button` in `portico-signin.css` already control the CSS of this element. I think you can set the opcaity there instead of creating a new ID here.

Also I guess this was supposed to go in the previous commit right?